AsyncDisplayKitTests/ASXCTExtensions.h

```diff
@@ -0,0 +1,37 @@
+//
+//  ASDisplayNodeLayoutTests.m
+//  AsyncDisplayKit
+//
+//  Copyright (c) 2014-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under the BSD-style license found in the
+//  LICENSE file in the root directory of this source tree. An additional grant
+//  of patent rights can be found in the PATENTS file in the same directory.
+//
```

Comment:
@maicki Let's yank this header stuff out, and move the stuff below up here. No need to be formal about landing it, feel free to just push to master